terraform/03_gke_cluster.tf

```diff
@@ -42,6 +42,7 @@ resource "random_shuffle" "zone" {
 resource "google_container_cluster" "gke" {
   provider = google-beta
   project = data.google_project.project.project_id
+  min_master_version = "1.16.13-gke.1"
```

Comment:
I just got an email about a Linux vulnerability in certain GKE versions - do we need to worry about this? This version is listed as an OK patch: 1.16.13-gke.401

Reply:
This version is stable. I think we're up to v1.18 now, so it's not a risky upgrade.

Reply:
We need to require one of these versions (that solve the vulnerability that was recently detected):

1.17.9-gke.1504
1.18.6-gke.3504

Reply:
Oh wait you guys are right, it has to be one of the following patches. Will upgrade and test things shortly 
1.16.13-gke.401
1.17.9-gke.1504
1.18.6-gke.3504